### PR TITLE
switching away from d2l-table to native table elements

### PR DESCRIPTION
**Pull request by dlockhart**
Table has recently been migrated to Lit, and the old `d2l-table`/`d2l-tr`/etc. elements are no longer needed now that IE11 is gone. This converts this repo over to just use the native table elements.

I've also migrated some of the other components over to use the core versions, which should have no effect.

### CODE REVIEW

File: d2l-activity-exemptions.js [hunk 1]
```diff
@@ -101,70 +99,45 @@ class D2LActivityExemptions extends mixinBehaviors(
...
 		</div>
 
-		<d2l-table
-		  id="classlist"
-		  summary="[[localize('ariaTableSummary')]]"
-		  aria-label$="[[localize('ariaTableSummary')]]"
-		>
-		  <d2l-thead>
-			<d2l-tr role="row">
-			  <d2l-th>
```
Comment (dlockhart) [inline d2l-activity-exemptions.js:111]: Probably easiest to look at this with whitespace disabled. Essentially though it's just a switch to native table elements, which means much of the ARIA roles and such can go away.

File: demo/index.html [hunk 2]
```diff
@@ -17,19 +17,17 @@
...
-		<div>
-			<h3>Basic activity-exemptions demo</h3>
-			<demo-snippet>
+		<h3>Basic activity-exemptions demo</h3>
+		<demo-snippet>
 			<template>
 				<d2l-activity-exemptions classlist-url="../testData/classlist.json" exemptions-url="../testData/exemptions.json">
 				</d2l-activity-exemptions>
+				<script>
+					setTimeout(() => document.querySelector('d2l-activity-exemptions').load());
```
Comment (dlockhart) [inline demo/index.html:28]: This is required now since an explicit call to `load()` is needed to kick things off.

**Review by davewagler-d2l — APPROVED** [review]

**Comment by davewagler-d2l** [comment]
I am no UI expert, but it looks good to me. I guess it works or it doesn't is probably obvious by visiting the page!

File: package.json [hunk 3]
```diff
@@ -10,14 +10,11 @@
   "author": "D2L Corporation",
   "license": "UNLICENSED",
   "dependencies": {
+    "@brightspace-ui/core": "^1",
     "@polymer/polymer": "^3.0.0",
-    "d2l-alert": "BrightspaceUI/alert#semver:^4",
```
Comment (davewagler-d2l) [inline package.json:14]: Pretty sweet, that we can get rid of all these by just including core.

Comment (dlockhart) [reply package.json:14]: Yep, almost everything has moved over there now!

**Pull request merged**